content/content.js

```diff
@@ -148,6 +147,7 @@ function loadForemHTML(forems) {
     document.getElementById('forem-sidecar') ||
     document.createElement('IFRAME');
   constructedSidecarIframe.id = 'forem-sidecar';
+  constructedSidecarIframe.title = 'My Forems';
```

Comment:
The actual fix. Open to suggestions for the name title of the iframe.

Reply:
Yeah we'll need to settle on the verbiage we want to use in the ecosystem at some point!